recipes/vulkan-loader/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "libvulkan-loader" %}
+{% set version = "1.3.239.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+# For this repo, v**** tags exists, but apparently the stable tags (the one corresponding to versions
+# that are available for download at https://vulkan.lunarg.com/sdk/home) are the sdk-*** ones
+# See https://github.com/KhronosGroup/Vulkan-Headers#version-tagging-scheme
+source:
+  url: https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/sdk-{{ version }}.tar.gz
+  sha256: fa2078408793b2173f174173a8784de56b6bbfbcb5fb958a07e46ef126c7eada
+
+build:
+  number: 0
+  ignore_run_exports_from:
+    - libxcb
```

Comment:
why?

Reply:
At least the version that is being packaged here was just using the headers of xcb, without linking the library, see https://github.com/KhronosGroup/Vulkan-Loader/blob/sdk-1.3.239.0/CMakeLists.txt#L180 . So keeping the `run_exports` was creating warnings (or an error? I can't remember) during the build. I noticed that this just changed in the latest master in https://github.com/KhronosGroup/Vulkan-Loader/pull/1136, so probably we will need anyhow to get rid of this in the next release.